app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= full_title(yield(:title)) %></title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all',
```

Comment:
- Sử dụng " thay cho ' trong toàn bộ file này
- Sau stylesheet_link_tag chỉ để 1 space thôi